airbyte-config/init/src/main/java/io/airbyte/config/init/RemoteConnectorCatalogPersistence.java

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Any particular reason you chose to add this as part of OSS and not directly in Cloud? (mainly just curious, with the proposed changes to the interface it might make more sense to keep this in OSS)